naming/src/main/java/com/alibaba/nacos/naming/controllers/ApiController.java

```diff
@@ -61,7 +62,7 @@ public JSONObject allDomNames(HttpServletRequest request) throws Exception {
         JSONObject result = new JSONObject();
         // For old DNS-F client:
         String dnsfVersion = "1.0.1";
-        String agent = request.getHeader("Client-Version");
+        String agent = request.getHeader(HttpHeaderConsts.USER_AGENT_HEADER);
```

Comment:
Or keep the original logic, is proposed, compatible with the old client

Reply:
I submitted some code that is compatible with older versions of the client.